test/commands/code-analyzer/rules.test.ts

```diff
@@ -200,35 +200,38 @@ describe('`code-analyzer rules` tests', () => {
 		});
 	});
 
-	describe('--workspace', () => {
+	describe.each([
+		{flag: '--workspace', shortflag: '-w', property: 'workspace'},
+		{flag: '--target', shortflag: '-t', property: 'target'},
```

Comment:
Since `--target` and `--workspace` share all the same constraints and expectations on a flag-level, I just made them variants of the same `describe.each` call.